### ISSUE TITLE
any reason why some of these extensions are listed as normative while other are informative?

### ISSUE THREAD

**Issue by boucadair** [ietf-wg-alto/draft-ietf-alto-oam-yang#75]
https://github.com/ietf-wg-alto/draft-ietf-alto-oam-yang/blob/57b228c241039dca16753625b71ee982b63dd1f8/introduction.md?plain=1#LL31C1-L32C58

**Comment by QiufangMa** [comment]
It seems to me that all should be listed as normative. 
Should RFC8341(NACM) be declared as informative?

**Comment by boucadair** [comment]
> Should RFC8341(NACM) be declared as informative?

No. Please see the note in https://wiki.ietf.org/group/ops/yang-security-guidelines. Thanks.

**Comment by fno2010** [comment]
> It seems to me that all should be listed as normative.

Agree. And I think `{{I-D.ietf-alto-performance-metrics}}` should also be normative.

**Comment by fno2010** [comment]
Does this commit ([662feec](https://github.com/ietf-wg-alto/draft-ietf-alto-oam-yang/commit/662feec36413cac1ee09dc0162ed13422d76f850)) fix this issue? Do we still miss anything?

**Comment by boucadair** [comment]
> And I think `{{I-D.ietf-alto-performance-metrics}}` should also be normative.

For this one, things are different as we do have the following from rfc8407:

     If a YANG module
      contains reference or "description" statements that refer to an
      I-D, then the I-D is included as an informative reference.

**Comment by boucadair** [comment]
> Does this commit ([662feec](https://github.com/ietf-wg-alto/draft-ietf-alto-oam-yang/commit/662feec36413cac1ee09dc0162ed13422d76f850)) fix this issue? Do we still miss anything?

I guess so. 

I just checked https://www.rfc-editor.org/cluster_info.php?cid=C453, and I think that it is safe to move "I-D.ietf-alto-performance-metrics" to be listed as normative.
